test/sync/src/commonMain/kotlin/io/realm/test/mongodb/util/AdminApi.kt

```diff
@@ -54,11 +61,15 @@ interface AdminApi {
     /**
      * Creates a remote user. Only call this function when `EmailPasswordAuth.registerUser` doesn't
      * make sense to use.
+     *
+     * Warning: This will run using `runBlocking`.
      */
     fun createUserRestApi(email: String, password: String)
```

Comment:
Should these just go now that we have the email password auth provider? Or does it still make sense to keep them around?

Reply:
This method is no longer being used, so I removed it.